File: VeryGoodViroGame/src/VeryGoodViroGame/TestCase.java
package VeryGoodViroGame;

import java.util.ArrayList;
import java.util.List;
import java.util.HashMap;

interface TestCaseRun {
    void run();
}

public class TestCase
{
    TestCaseRun mRun;
    String name;
    String desc;

    public TestCase(String name, String desc, TestCaseRun mRun)
    {
        this.name = name;
        this.desc = desc;
        this.mRun = mRun;
    }
}

File: tervezes/uml_gen/uml_generator/TestCase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace VeryGoodViroGame
{
    interface ITestCaseRun
    {
        void Run();
    }

    public class TestCase
    {
        TestCaseRun mRun;
        string name;
        string desc;
        public TestCase(string name, string desc, TestCaseRun mRun)
        {
            this.name = name;
            this.desc = desc;
            this.mRun = mRun;
        }
    }
}